testData/solutions/win32.json5

```diff
@@ -16206,5 +16206,92 @@
                 "type": "gpii.deviceReporter.alwaysInstalled"
             }
         ]
+    },
+    "net.gpii.morphic": {
+        "name": "Morphic",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.settingsHandlers.noSettings",
```

Comment:
Doesn't this actually use the WebSockets settings handler?

Reply:
I feel like this probably uses a settings handler that doesn't quite exist yet, which would include whatever code in the gpii-app would re-arrange it's UI when a key-in occurs and these new settings come down from the flowManager.

Reply:
Hey @sgithens, actually, this settings handler already exists (see https://github.com/GPII/universal/blob/master/documentation/BrowserChannel.md) and is what UIO+ uses to receive settings from the local flowManager (see https://github.com/GPII/universal/blob/master/testData/solutions/win32.json5#L12421). You can follow the UIO+ example for the Morphic QSS solution registry entry.
Regarding the gpii-app side, I guess that we should consult HA about the best way to make it happen.